### PR TITLE
Feature: tsne

### PR DESCRIPTION
**Pull request by maxhutch**
Adding t-SNE call that hits the data analysis endpoint and then re-formats the keys a bit.

CC: @eantono thoughts?

### CODE REVIEW

File: citrination_client/client.py [hunk 1]
```diff
@@ -304,6 +304,35 @@ def upload_file(self, file_path, dataset_id, root_path=None):
         """
         return self.upload(dataset_id, file_path)
 
+    def _data_analysis(self, model_name):
```
Comment (mcgalcode) [inline citrination_client/client.py]: I think nearly this exact method exists in `feature/dev-api` as a public method (and is used in the api tests). Should we keep this private or let users consume data analyses directly?

File: citrination_client/client.py [hunk 2]
```diff
@@ -516,3 +545,10 @@ def _get_create_data_set_version_url(self, data_set_id):
         :return: URL for creating new data set versions.
         """
         return self.api_url+'/data_sets/'+str(data_set_id)+'/create_dataset_version'
+
+
+    def _get_data_analysis_url(self, model_name):
+        return self.api_url + '/data_views/' + model_name + '/data_analysis'
+
+    def _get_estimators_url(self, model_name):
+        return self.api_url + '/data_views/' + model_name + '/estimators'
```
Comment (mcgalcode) [inline citrination_client/client.py]: Is this related to the other content in this PR somehow?

Comment (maxhutch) [inline citrination_client/client.py]: No; this bled through.  Removing.

File: citrination_client/client.py (hunk 1 above)
Comment (maxhutch) [inline citrination_client/client.py]: I don't think the raw output from the dev-api is user friendly, which is why I "hid" it.  In a future PR, we can expose the correlations as another method.

**Comment by eantono** [comment]
The dictionary format looks reasonable to me :+1:

File: citrination_client/client.py (hunk 1 above)
Comment (mcgalcode) [inline citrination_client/client.py]: Sounds good to me

File: citrination_client/client.py (hunk 2 above)
Comment (mcgalcode) [inline citrination_client/client.py]: 👍
